drivers/srf04/include/srf04_params.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_srf04
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for srf04 devices
+ *
+ * @author      Semjon Kerner <semjon.kerner@fu-berlin.de>
+ */
+
+#ifndef SRF04_PARAMS_H
+#define SRF04_PARAMS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "board.h"
+#include "srf04.h"
+
+/**
+ * @name   Default configuration parameters for SRF04 device
+ * @{
+ */
+#ifndef SRF04_PARAM_TRIGGER
+#define SRF04_PARAM_TRIGGER     GPIO_PIN(0,13)
+#endif
+#ifndef SRF04_PARAM_ECHO
+#define SRF04_PARAM_ECHO        GPIO_PIN(0,14)
+#endif
+
+#ifndef SRF04_PARAMS
+#define SRF04_PARAMS                            \
+    {   .state          = SRF04_IDLE,           \
+        .trigger        = SRF04_PARAM_TRIGGER,  \
+        .echo           = SRF04_PARAM_ECHO,     \
+        .distance       = (0U),                 \
```

Comment:
Why do you need distance in the initialization parameters ? Same question for time and I also saw below that state is also added in the parameter struct. I think those 3 should go in the device descriptor.